intermediate_source/dist_pipeline_parallel_tutorial.rst

```diff
@@ -74,9 +74,16 @@ the two ResNet shards.
         return nn.Conv2d(in_planes, out_planes, kernel_size=1, stride=stride, bias=False)
 
 
+    def get_cuda_if_available(i):
+        assert i >= 0
+        if torch.cuda.is_available():
+            return f"cuda:{min(i, torch.cuda.device_count() - 1)}"
+        else:
+            return "cpu"
```

Comment:
similar to my comment on the PR in the example repo, since we intended to show the benefit of pipeline parallelism, this tutorial requires at least two GPUs.